core/src/main/java/org/apache/calcite/rel/RelDistributions.java

```diff
@@ -38,19 +38,14 @@
 public class RelDistributions {
   public static final ImmutableIntList EMPTY = ImmutableIntList.of();
 
-  /** The singleton singleton distribution. */
```

Comment:
I guess this is not a 'typo'. You can see other comments below, the first 'singleton' means the design pattern is 'singleton', and the second 'singleton' means the 'singleton distribution'.